storage/src/cache/cachedfile.rs

```diff
@@ -447,20 +447,20 @@ impl BlobObject for FileCacheEntry {
             if let Some(meta) = self.meta.as_ref() {
                 if let Some(bm) = meta.get_blob_meta() {
                     if let Some(chunks) = bm.add_more_chunks(chunks, max_size) {
-                        return self.do_fetch_chunks(&chunks);
+                        return self.do_fetch_chunks(&chunks, true);
                     }
                 } else {
                     return Err(einval!("failed to get blob.meta"));
                 }
             }
         }
 
-        self.do_fetch_chunks(chunks)
+        self.do_fetch_chunks(chunks, true)
     }
 }
 
 impl FileCacheEntry {
-    fn do_fetch_chunks(&self, chunks: &[BlobIoChunk]) -> Result<usize> {
+    fn do_fetch_chunks(&self, chunks: &[BlobIoChunk], prefetch: bool) -> Result<usize> {
```

Comment:
The newly added argument `prefetch` is only for debugging purposes? Can we remove it?

Reply:
It may help to analyze online issues, so added it:)